logical/framework/backend.go

```diff
@@ -506,6 +506,10 @@ type FieldSchema struct {
 	Required    bool
 	Deprecated  bool
 
+	// Query indicates this field will be sent as a query parameter. It doesn't
+	// affect handling of the value, but may be used for documentation.
+	Query bool
```

Comment:
Would it be possible to add an example here or in a test somewhere? Maybe it's because I'm context-switching into reading this code, but I don't totally "get" how a query parameter would be used and appear.

Reply:
The additional test case is here: https://github.com/hashicorp/vault/pull/6490/files#diff-ce9d761e86729f4a2074481f131e60baR351

But I'll improve the description here. Query is a very overloaded term, so better to be explicit.